README.md

```diff
@@ -11,93 +11,124 @@ Master (Release): [![Build Status](https://jenkins.qa.cancercollaboratory.org/bu
 
-### Quickstart
+- [Getting Started](#getting-started)
+  - [Development Setup](#--development-setup)
+  - [Dockerized Setup (a.k.a. Quickstart)](#--dockerized-setup)
+- [Motivation](#motivation)
+- [What is a "Data Portal"?](#data-portal)
+- [Roadmap](#roadmap)
+- [Development Details](#development-details)
 
-#### Clone, Install, Bootstrap
+---
 
-```
-git clone git@github.com:overture-stack/arranger.git
-cd arranger
-npm i
-npm run bootstrap
+### Getting Started
 
-# watch modules and rebuild on change
-npm run watch
+#### - Development Setup
 
-# run tests for all modules
-npm test
+Setting up the project, and ready things to make changes
 
-# run server on :5050
-npm run server
+```bash
+# 1. clone the repository
+  git clone git@github.com:overture-stack/arranger.git
+
+# 2. enter the project's folder
+  cd arranger
 
-# run dashboard on :6060
-npm run dashboard
+# 3. install the dependencies
+  npm i
 
-# run portal on :7070
-npm run portal
+# 4. install the module's own dependencies
+  npm run bootstrap
 
-# run storybook on :8080
-npm run storybook
 ```
 
-- [Documentation](#Documentation)
-- [Motivation](#motivation)
-- [What is a "Data Portal"?](#data-portal)
-  - [Topology](#topology)
-- [Roadmap](#roadmap)
-- [Development](#development)
-- [License](https://github.com/overture-stack/arranger/blob/master/LICENSE)
+Now you should be able to start the following processes from the project's root:
 
-#### Docker
+```bash
+# watch all modules and rebuild them when you make changes
+  npm run watch
 
-##### Start all services
+# test all modules at once
+  npm test
 
-```bash
-make start
-```
+# run the server (on port 5050)
+  npm run server
 
-##### Bootstrap the file_centric index
+# serve the component dashboard (on port 6060)
+  npm run dashboard
 
-```bash
-make init-es
+# serve the component portal (on port 7070)
+  npm run portal
+
+# run storybook (on port 8080)
+  npm run storybook
 ```
 
-##### Show other make targets
+#### - Dockerized Setup
+
+A bit more friendly "quickstart", if you just want to get things started
 
 ```bash
-make help
-```
+# Start all services at once, using some default settings.
+# This runs the following services: Elasticsearch, kibana, arranger-server, and arranger-ui
+  make start
+
+# ^^^ which runs the following command behind the scenes:
+# ES_USERNAME=elastic ES_PASSWORD=myelasticpassword docker-compose -f docker-compose.yml up -d -build
+# Note: these ES_* values may be customized when running your own Arranger instance
 
-### Documentation
 
-For in-depth documentation, please see the Arranger Read the Docs: https://arranger.readthedocs.io/en/latest/
+---
+# Afterwards, in another bash process, you may seed an example file_centric index
+  make init-es
+
+# ^^^ which runs the following command behind the scenes:
+# ./docker/elasticsearch/load-es-data.sh ./docker/elasticsearch elastic myelasticpassword
+# That SH script may give you ideas on how to automate uploading indexes to your instance.
```

Comment:
Just to clarify, this is additional enhanced scripting they can do themselves correct?  But not necessary/mandatory for base install?  (i.e. I don't they don't necessarily need to write a script to automate correct?  i.e. they don't need to write their own Maestro :) )

Reply:
That's correct. This was just a pointer, for those who want to seed a different set of scaffoldings, or are not using maestro at all.